src/DoctrineMigrations/Version20210910210740.php

```diff
@@ -0,0 +1,113 @@
+        $this->addSql('CREATE INDEX IDX_12AD233ECD678BED ON issue (content_item_id)');
+        $this->addSql('CREATE INDEX IDX_12AD233EC38008F2 ON issue (fixed_by_id)');
+        $this->addSql('CREATE TABLE log_entry (id INT NOT NULL, user_id INT DEFAULT NULL, course_id INT DEFAULT NULL, message TEXT DEFAULT NULL, severity VARCHAR(255) NOT NULL, created TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, status BOOLEAN DEFAULT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE INDEX IDX_B5F762DA76ED395 ON log_entry (user_id)');
+        $this->addSql('CREATE INDEX IDX_B5F762D591CC992 ON log_entry (course_id)');
+        $this->addSql('CREATE TABLE report (id INT NOT NULL, course_id INT NOT NULL, author_id INT NOT NULL, data TEXT DEFAULT NULL, created TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, errors INT DEFAULT NULL, suggestions INT DEFAULT NULL, ready BOOLEAN NOT NULL, content_fixed INT DEFAULT NULL, content_resolved INT DEFAULT NULL, files_reviewed INT DEFAULT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE INDEX IDX_C42F7784591CC992 ON report (course_id)');
+        $this->addSql('CREATE INDEX IDX_C42F7784F675F31B ON report (author_id)');
+        $this->addSql('CREATE TABLE user_session (id INT NOT NULL, uuid VARCHAR(255) NOT NULL, data JSON DEFAULT NULL, created TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE TABLE users (id INT NOT NULL, institution_id INT NOT NULL, username VARCHAR(180) NOT NULL, name VARCHAR(255) DEFAULT NULL, roles JSON DEFAULT NULL, lms_user_id VARCHAR(128) NOT NULL, api_key VARCHAR(2048) DEFAULT NULL, refresh_token VARCHAR(512) DEFAULT NULL, created TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, last_login TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_1483A5E9F85E0677 ON users (username)');
+        $this->addSql('CREATE INDEX IDX_1483A5E910405986 ON users (institution_id)');
+        $this->addSql('CREATE TABLE messenger_messages (id BIGSERIAL NOT NULL, body TEXT NOT NULL, headers TEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, available_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, delivered_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE INDEX IDX_75EA56E0FB7336F0 ON messenger_messages (queue_name)');
+        $this->addSql('CREATE INDEX IDX_75EA56E0E3BD61CE ON messenger_messages (available_at)');
+        $this->addSql('CREATE INDEX IDX_75EA56E016BA31DB ON messenger_messages (delivered_at)');
+        $this->addSql('CREATE OR REPLACE FUNCTION notify_messenger_messages() RETURNS TRIGGER AS $$
```

Comment:
Postgres really needs all of these changes in order to work?

Reply:
Yes Postgres does need these to work. The code could be condensed with psuedo-classes (e.g., `SERIAL` for the ID incrementing) though, if that sounds desirable.